assets/dev/js/editor/regions/navigator/component.js

```diff
@@ -1,7 +1,28 @@
 import ComponentBase from 'elementor-api/modules/component-base';
+import ElementsComponent from './elements/component';
+
 import * as commands from './commands/';
+import * as hooks from './commands-hooks';
+import * as commandsInternal from './commands-internal/';
 
 export default class Component extends ComponentBase {
+	isDocked = false;
+
+	__construct( args ) {
+		super.__construct( args );
+
+		this.region = args.manager;
+
+		this.elements = $e.components.register( new ElementsComponent( { manager: this } ) );
+
+		Object.defineProperty( this, 'indicators', {
+			get() {
+				elementorCommon.helpers.softDeprecated( 'elementor.navigator.indicators', '3.6.0', 'elementor.navigator.region.indicators' );
```

Comment:
Replace in all files

```suggestion
				elementorCommon.helpers.softDeprecated( 'elementor.navigator.indicators', '3.8.0', 'elementor.navigator.region.indicators' );
```

Reply:
See: https://github.com/elementor/elementor/pull/17938